lib/puma/server.rb

```diff
@@ -227,6 +227,9 @@ def handle_servers_lopez_mode
     # are handled synchronously.
     #
     def run(background=true)
+      @max_threads = Integer(@max_threads)
```

Comment:
`@max_threads` and `@min_threads` are strings at this point if specified in `@options`. It's strange because `workers` is an integer... We can think about how to improve this later.